docs/Preparing_your_workstation.adoc

```diff
@@ -57,58 +59,58 @@ The lint tools are optional but are recommended tools to check the quality of yo
 ----
 
 # Install basic packages
-yum install -y  wget python python-boto unzip python2-boto3.noarch tmux git ansible
+$ sudo yum install -y  wget python python-boto unzip python2-boto3.noarch tmux git ansible
 
 # Another option to configure python boto is:
-git clone git://github.com/boto/boto.git
-cd boto
-python setup.py install
+$ git clone git://github.com/boto/boto.git
+$ cd boto
+$ python setup.py install
 
 #Install boto3
-pip install boto3
+$ pip install boto3
 
 #Install pywinrm if you plan to deploy windows VMs
 #pip install pywinrm
 
 # Enable epel repositories for Ansible
-cd /tmp
-wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-sudo yum -y install `ls *epel*.rpm`
+$ cd /tmp
+$ wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
I'm not in favor of using EPEL for anything that is not absolutely necessary. What is the need for it here to prepare your workstation? I think this might be legacy documented steps since you can easily get ansible via pip now.

Reply:
I agree. I actually think whilst it is more work we may need 3 "workstation" docs linked from a master: RHEL7, RHEL8/Fedora<x>, and Mac. Plus it may be better to be opinionated e.g. python3, pip for boot etc and I think we may want to include a requirements file for your "workstation" (archaic term?). @wilson-walrus you've also been looking at this?

Reply:
I am also in favor of removing EPEL where not 100% needed. 

I think having a separate path for the various operating systems is a great idea too. I also think it would be beneficial to have a master list required packages for people what will want to install in their own way.

Reply:
@wilson-walrus @amayagil to discuss separately in a call before merging or closing. After all we are not using this doc for the training.